CHANGELOG.md

```
@@ -4,143 +4,447 @@ All notable changes to LeedPDF will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## [Unreleased]
+---
+
+## [v2.31.0] - 2026-03-27
 
 ### Added
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)